cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -447,7 +447,17 @@ values for Cloudera may vary from the above appendix:
     :start-after: .. _cloudera-starting-services-java-heapmax:
     :end-before: .. end_of_list
 
-At this point, the CDAP installation is configured and is ready to be installed. Review
+**The CDAP system user:** As Hadoop resolves users at the NameNode, the ``cdap`` user must
```

Comment:
No, that makes sense: I've moved it up to under `Preparing the Cluster > Create the "cdap" User`.